### ISSUE TITLE
Cannot add custom walk paths in standalone version

### ISSUE THREAD

**Issue by Dream-Cypher** [zanllp/sd-webui-infinite-image-browsing#452]
Hello,
In the sd-webui extension, I can add additonal paths to the "walk" menu, for example  d:/otherapp/output.

On the standalone exe version, it warns me that this could take a long time to index, but even when I click "OK", and it says it was successful, the path is not added to the menu.

**Comment by zanllp** [comment]
Is there any error message?

**Comment by zanllp** [comment]
Is this folder located on a network hard drive? IIB doesn't support some locally mounted network hard drives very well.

**Comment by Dream-Cypher** [comment]
No, just folders on the local hard drive.  I am able to add them using the extension from within A1111, just not using the standalone version.

**Comment by yellowcooln** [comment]
> IB doesn't support some locally mounted network hard drives very well.

That does explain why it's not mounting for me. I wanted to mount an old outputs folder from my network share and it didn't work. I can view files from the current outputs folder with no issues, I just can't mount other ones.

![image](https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/12516003/f1bf22d9-1bf9-4a6c-87e2-ca72a3f733d4)

**Comment by Rakly3** [comment]
in the `sd-webui-infinite-image-browsing` folder locate the file `.env.example`
Make a copy of it in the same folder and rename it to `.env`

Open the .env file with notepad and change
`IIB_ACCESS_CONTROL=auto` to `IIB_ACCESS_CONTROL=disable`
Yes, that's `disable`

Relaunch the app/A1111 and add your custom path.
_I have not tested this with network mounted shares._

**Comment by yellowcooln** [comment]
> Open the .env file with notepad and change
> IIB_ACCESS_CONTROL=auto to IIB_ACCESS_CONTROL=disable
> _I have not tested this with network mounted shares._

I can confirm it works!!! 


![image](https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/12516003/f0e98b8a-c741-4528-ac28-fff6c87fb124)

_No, the 40TB is not all StableDiffusion Outputs. It's just what I named the share in Unraid._

![image](https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/12516003/c92c8864-4735-4e38-82a3-8a8a5760e0d1)
![image](https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/12516003/486485aa-d574-420b-a325-04cb22b088bf)
![image](https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/12516003/f4de4991-28a2-48b4-afd5-1ad4783c65a4)
